.github/workflows/image.yaml

```diff
@@ -180,6 +180,26 @@ jobs:
             ./earthly.sh +datasource-iso --CLOUD_CONFIG=tests/assets/autoinstall.yaml
             ./earthly.sh +run-qemu-datasource-tests --FLAVOR=${{ matrix.flavor }}
 
+  qemu-install-tests:
+    needs:
+    - build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+       include:
+         - flavor: "alpine-opensuse-leap"
```

Comment:
maybe just focus on one flavor only? I don't see anything that should impact this at distro-level (and would also keep a lot of cycles of our CI busy)

Reply:
Sure, we already removed a lot. I agree it's enough to use only one favor.